app/src/main/java/org/imozerov/streetartview/ui/extensions/MapExtensions.kt

```diff
@@ -56,7 +64,7 @@ fun GoogleMap.addUserLocationMarker(userLocation: LatLng) {
 
 fun GoogleMap.addArtObject(artObject: ArtObjectUi) {
     val markerOptions = MarkerOptions().position(LatLng(artObject.lat, artObject.lng))
-            .title(artObject.authorsNames())
+            .title(artObject.id)
```

Comment:
So user will be able to see art object id by clicking on the marker? If so - it doesn't seem right.